cmake failure when using ZEPHYR_MODULES without west

**Describe the bug**
 I am getting below compilation error for  `bcm958402m2_m7` board on latest zephyr:

```
CMake Error at /home/zephyr/cmake/zephyr_module.cmake:45 (message):
  Traceback (most recent call last):
    File "/home/zephyr/scripts/zephyr_module.py", line 275, in <module>
      main()
    File "/home/zephyr/scripts/zephyr_module.py", line 201, in main
      projects += args.extra_modules
  UnboundLocalError: local variable 'projects' referenced before assignment

```
**To Reproduce**
`cmake -GNinja -B build/ -DBOARD=bcm958402m2_m7 -DZEPHYR_MODULES=<PATH TO hal/cmsis/ directory> samples/hello_world/
`

**Expected behavior**
This issue seems to be introduced in commit ef3c5e551676278ccc1f42ecb572838144c5e33a (as pointed by @nashif ).

--- a/scripts/zephyr_module.py
+++ b/scripts/zephyr_module.py
@@ -197,6 +197,8 @@ def main():
             # workspace. Such setup is allowed, as west may be installed
             # but the project is not required to use west.
             projects = []
+    else:
+        projects = args.modules.copy()
 
     projects += args.extra_modules
     extra_modules = set(args.extra_modules)

